crates/goose-mcp/src/developer/mod.rs

```diff
@@ -312,9 +312,9 @@ impl DeveloperRouter {
 
     async fn text_editor_view(&self, path: &PathBuf) -> Result<Vec<Content>, ToolError> {
         if path.is_file() {
-            // Check file size first (2MB limit)
-            const MAX_FILE_SIZE: u64 = 2 * 1024 * 1024; // 2MB in bytes
-            const MAX_CHAR_COUNT: usize = 1 << 20; // 2^20 characters (1,048,576)
+            // Check file size first (400KB limit)
+            const MAX_FILE_SIZE: u64 = 400 * 1024; // 400KB in bytes
```

Comment:
let's recreate the same limit on shell tool outputs? and same warning/error

Reply:
done